src/nvim/mouse.c

```diff
@@ -682,39 +682,38 @@ static int mouse_adjust_click(win_T *wp, int row, int col)
 
   vcol = offset;
 
+#define incr() nudge++; ptr_end += utfc_ptr2len(ptr_end)
+#define decr() nudge--; ptr_end -= utfc_ptr2len(ptr_end)
```

Comment:
while we're here, they should be uppercase

```c
#define INCR() nudge++; ptr_end += utfc_ptr2len(ptr_end)
#define DECR() nudge--; ptr_end -= utfc_ptr2len(ptr_end)

Reply:
Forgot. Thanks for the reminder.